### PR TITLE
added text of 4-3-20 proposal

### PR DESCRIPTION
**Pull request by unchainedio**
added the text of the proposal from DC to Github

### CODE REVIEW

**Review by WalterSobchakDash — APPROVED** [review]
I'm happy to approve this.

**Comment by ColeDIF** [comment]
This is a comment.

File: 4-13-2020.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Dash Investment Foundation: April - June funding (3 months)
+
+Monthly ask will be 500 Dash for the next 3 months.
+
+In line with all previous proposals we are looking for continued funding of 500D per month.
```
Comment (essra-one) [inline 4-13-2020.md:5]: Is this how I submita comment?

**Review by ColeDIF — APPROVED** [review]
approved

**Review by ColeDIF — APPROVED** [review]
re-approved

**Review by essra-one — APPROVED** [review]
This is exactly what we agreed on

**Review by kodaxx — APPROVED** [review]
Approval

**Pull request merged**